sale_payment_sheet/models/sale_payment_sheet.py

```diff
@@ -183,7 +182,6 @@ def button_reopen(self):
             raise UserError(
                 _("You can not reopen a sheet that has any reconciled line.")
             )
-        self_sudo.statement_id.button_reopen()
```

Comment:
Why this change?

Reply:
button_reopen() does not exist anymore. I add unlink statement as it is on v15.